op-deployer/pkg/deployer/integration_test/apply_test.go

```diff
@@ -613,6 +619,126 @@ func TestApplyGenesisStrategy(t *testing.T) {
 	}
 }
 
+func TestProofParamOverrides(t *testing.T) {
+	op_e2e.InitParallel(t)
+
+	lgr := testlog.Logger(t, slog.LevelDebug)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	depKey := new(deployerKey)
+	l1ChainID := big.NewInt(77799777)
+	dk, err := devkeys.NewMnemonicDevKeys(devkeys.TestMnemonic)
+	require.NoError(t, err)
+
+	l2ChainID1 := uint256.NewInt(1)
+
+	deployerAddr, err := dk.Address(depKey)
+	require.NoError(t, err)
+
+	loc := localArtifactsLocator(t)
+
+	env, bundle, _ := createEnv(t, ctx, lgr, nil, broadcaster.NoopBroadcaster(), deployerAddr)
+	intent, st := newIntent(t, l1ChainID, dk, l2ChainID1, loc, loc)
+	intent.Chains = append(intent.Chains, newChainIntent(t, dk, l1ChainID, l2ChainID1))
+	intent.DeploymentStrategy = state.DeploymentStrategyGenesis
+	intent.GlobalDeployOverrides = map[string]any{
+		"withdrawalDelaySeconds":                  standard.WithdrawalDelaySeconds + 1,
+		"minProposalSizeBytes":                    standard.MinProposalSizeBytes + 1,
+		"challengePeriodSeconds":                  standard.ChallengePeriodSeconds + 1,
+		"proofMaturityDelaySeconds":               standard.ProofMaturityDelaySeconds + 1,
+		"disputeGameFinalityDelaySeconds":         standard.DisputeGameFinalityDelaySeconds + 1,
+		"mipsVersion":                             standard.MIPSVersion + 1,
+		"disputeGameType":                         standard.DisputeGameType, // This must be set to the permissioned game
+		"disputeAbsolutePrestate":                 common.Hash{'A', 'B', 'S', 'O', 'L', 'U', 'T', 'E'},
+		"disputeMaxGameDepth":                     standard.DisputeMaxGameDepth + 1,
+		"disputeSplitDepth":                       standard.DisputeSplitDepth + 1,
+		"disputeClockExtension":                   standard.DisputeClockExtension + 1,
+		"disputeMaxClockDuration":                 standard.DisputeMaxClockDuration + 1,
+		"dangerouslyAllowCustomDisputeParameters": true,
```

Comment:
I like this, and wonder if we should do something similar for the L1 and L2 contract locator tags also, to discourage changing those

Reply:
What are you envisioning for the tags here?

Reply:
Basically just want to make it very unlikely that people don't use the standard default releases (1.6.0 for L1 and 1.7.0 beta for L2). Perhaps even removing those two locator tags from the default intent to make that option less noticeable

Reply:
Ah gotcha! Yes I can modify the locators to be overrides rather than setting them raw.